### DISCUSSION TITLE
Preventing east/west navigation from wrapping around the world

### DISCUSSION THREAD

**Discussion by mlidal** [Leaflet/Leaflet#9359] [Q&A] [answered]
Hello. Is there any way in leaflet to prevent users from scrolling the map around the world when panning east or west? (Also prevent users from zooming far enough out to show multiple maps at the same time?)

**Comment by IvanSanchez** [comment, marked as answer]
https://leafletjs.com/reference.html#map-maxbounds

**Reply by mlidal** [reply]
Thanks :)
